static/js/popovers.js

```diff
@@ -1107,27 +1107,61 @@ exports.register_click_handlers = function () {
     }
 
     $("body").on("click", ".remind.in_20m", (e) => {
-        const datestr = moment().add(20, "m").format();
+        const datestr = formatISO(
+            add(new Date(), {
+                minutes: 20,
+            }),
+        );
```

Comment:
The one-line style seems more appropriate for these functions. (Prettier accepts either.)
```js
const datestr = formatISO(add(new Date(), {minutes: 20}));
```